houston/ardu/command_factory.py

```diff
@@ -27,6 +46,7 @@ def create_command(command: Dict[str, Any]) -> Type[Command]:
     except KeyError:
         msg = "missing 'id' field of Command"
         raise TypeError(msg)
+    generator = command.get('generator')
```

Comment:
What happens if a command doesn't have a generator? `generator` will be `None`, of course, but how will that affect the rest of the system?